hackcon-2015/README.md

```diff
@@ -0,0 +1,27 @@
+# hackcon15 write-ups
+
+* [Website](http://hackcon.in/)
+* [Scoreboard](http://hackcon.in/scoreboard)
+
```

Comment:
give us a permanent score board

ie it should be valid even after you guys put down the server or conduct a new one next year